compensated-ruby/lib/compensated/gumroad/event_parser.rb

```diff
@@ -9,27 +9,44 @@ def parses?(request)
         keys.include?(:seller_id) && keys.include?(:product_id) && keys.include?(:product_permalink) && request.data["product_permalink"].include?("gum.co")
       end
 
-      def normalize(data_or_body)
-        data = data_or_body.respond_to?(:key) ? data_or_body : data_from_string(data_or_body)
+      def transform(data_or_body)
```

Comment:
I was trying to write some doc for this but didn't know what's the convention if the params could be two different data types, do you know what's a good way to document this?

Reply:
I believe that Ruby doc `@param` attribute can accept multiple types:

```
@param data_or_body [IO, String,Rack::Request] Data from Gumroad
```